VGC/models/User.js

```diff
@@ -25,6 +25,9 @@ const UserSchema = new mongoose.Schema({
         type: String,
         default: 'User'
     },
+    wishlist: [{
+        type: String
+    }]
```

Comment:
Here your array is an list of string, but it seems that you are storing the gameID which might be a number rather than a string. Check on this to make sure the type is correct.